### PR TITLE
Fix issue with casting of FindResponse

### PR DESCRIPTION
**Pull request by linuxchata**
# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

Insert a short (one or two sentence) summary here.

Fixes #Issue_Number

>Note: The "Fixes #nnn" syntax in the PR description causes
>GitHub to automatically close the issue when this PR is merged.
> Remove that line if you don't have issues associated with this
> PR. Click on the Guidelines for Contributing link above for details.

## Details

Explain your changes, and why you made them. If that
information is already available in the issue referenced
above, just referencing the issue is preferred to copying
the text.

This may not be necessary depending on the scope of the PR 
changes. For example, "fix typo in introduction.md" is
sufficient to describe that PR.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

File: docs/framework/wcf/feature-details/wcf-discovery-overview.md [hunk 1]
```diff
@@ -115,39 +115,39 @@ class Client
...
+    }  
+  
+    static void InvokeService()  
+    {  
+        Console.WriteLine("\nInvoking Calculator Service at {0}\n", serviceAddress);  
+  
+        // Create a client  
+        CalculatorClient client = new CalculatorClient();  
+        client.Endpoint.Address = serviceAddress;  
+        client.Add(10,3);
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/wcf-discovery-overview.md]: An `}` is missing to terminate the method implementation.

File: docs/framework/wcf/feature-details/wcf-discovery-overview.md [hunk 2]
```diff
@@ -115,39 +115,39 @@ class Client
...
-}
+    // ** DISCOVERY ** //  
+    static bool FindService()  
+    {  
+        Console.WriteLine("\nFinding Calculator Service ..");  
+        DiscoveryClient discoveryClient =   
+            new DiscoveryClient(new UdpDiscoveryEndpoint());  
+  
+        Collection<EndpointDiscoveryMetadata> calculatorServices =   
+            (Collection<EndpointDiscoveryMetadata>)discoveryClient.Find(new FindCriteria(typeof(ICalculator)));
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/wcf-discovery-overview.md]: This cast still fails. `DiscoveryClient.Find` returns a `FindResponse` object; the code should retrieve its `FindResponse.Endpoints` property and cast that to a `Collection<EndpointDiscoveryMetadata>`.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for improving this code example, @linuxchata. However, the cast still does not succeed. Could you make the changes that I've suggested? Or, assuming that they are correct, we can make them for you if you'd prefer. 
Incidentally, in fixing a merge conflict, I also added language tags to the fenced code blocks.

File: docs/framework/wcf/feature-details/wcf-discovery-overview.md (hunk 2 above)
Comment (linuxchata) [reply docs/framework/wcf/feature-details/wcf-discovery-overview.md]: Absolutely correct that Endpoints is missing at the end of the expression. I have committed that change

**Comment by rpetrusha** [comment]
Thanks for making the change, @linuxchata. We'll merge your PR now. Your changes should appear live on docs.microsoft.com tomorrow.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**